example/vendor.ts

```diff
@@ -3,13 +3,6 @@ import '@angular/platform-browser';
 import '@angular/platform-browser-dynamic';
 import '@angular/core';
 
-import 'rxjs/add/observable/fromEvent';
-import 'rxjs/add/observable/merge';
-import 'rxjs/add/observable/of';
-import 'rxjs/add/operator/filter';
-import 'rxjs/add/operator/finally';
-import 'rxjs/add/operator/map';
-import 'rxjs/add/operator/sampleTime';
-import 'rxjs/add/operator/switchMap';
-import 'rxjs/add/operator/take';
-import 'rxjs/Observable';
+import { fromEvent, merge, of } from 'rxjs';
+import { filter, finalize, map, sampleTime, switchMap, take } from 'rxjs/operators';
+import { Observable } from 'rxjs';
```

Comment:
There is no reason to import these functions here.

Reply:
oops. should be removed now.